client/rest-high-level/src/test/java/org/opensearch/client/core/tasks/GetTaskResponseTests.java

```diff
@@ -41,6 +40,7 @@
 import org.opensearch.core.tasks.resourcetracker.TaskThreadUsage;
 import org.opensearch.core.xcontent.ToXContent;
 import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.internal.client.Requests;
```

Comment:
Same, could we get rid of `Requests` usage?

Reply:
For this class `Requests` is needed as it uses `Requests.INDEX_CONTENT_TYPE.xContent())`.

Reply:
I see what you mean, we should be able to directly import the statics.

Reply:
Or find other ways to detach these clients